dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -88,7 +88,8 @@ RUN pip install cffi==1.10.0 \
     && rm -fr nanomsg-1.0.0 \
     && rm -f 1.0.0.tar.gz  \
     && pip install nnpy    \
-    && pip install dpkt
+    && pip install dpkt    \
+    && pip install scapy --upgrade
```

Comment:
What is the relationship between this pip installed `scapy` and the apt installed `python-scapy` above?

Reply:
python-ptf will check whether python-scapy was installed, but the python-scapy is the previous version that cannot meet our requirement. So I install python-ptf firstly and then upgrade scapy by pip.